KubeArmor/monitor/systemMonitor.go

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+//Monitor package is the component responsible for monitoring syscalls
```

Comment:
```suggestion
// Package monitor is the component responsible for monitoring syscalls and communicating with eBPF Programs
```
Pleasing the linter again :D